fw/tls.c

```diff
@@ -833,7 +834,7 @@ tfw_tls_get_if_configured(TfwVhost *vhost)
 
 #define SNI_WARN(fmt, ...)						\
 	TFW_WITH_ADDR_FMT(&cli_conn->peer->addr, TFW_NO_PORT, addr_str,	\
-			  T_WARN("TLS: sni ext: client %s requested "fmt, \
+			  T_WARN("client %s requested"fmt, \
```

Comment:
I'm wondering what might be the reason for the broken last octet in the client IP `104.131.181.:/` as cited in #1511 ? Was it some formatting issue or did we experience a broader memory corruption issue?

Reply:
The reason was broken `tfw_put_dec()` and we have fixed that in #1519. Here I just shorten it because it's clear that all 3 messages containing vhost is about sni.